src/actions/packaging.rs

```diff
@@ -156,7 +156,7 @@ pub fn packages_stage_select<'a, K: Clone + ExactSizeIterator<Item = &'a str>>(
         packages
             .iter()
             .position(|x| {
-                x == start_package || x.splitn(2, '/').next().unwrap_or("") == start_package
+                x == start_package || x.splitn(2, '/').collect::<Vec<_>>()[1] == start_package
```

Comment:
This would produce unexpected panic if the package name passed here does not contain `/` symbol.